### ISSUE TITLE
Old Macroinvertabrate Datasheets do not calculate Tolerant Totals Correctly

### ISSUE THREAD

**Issue by hertelc** [osuosl/streamwebs#541]
When I add a new Macroinvertebrate datasheet I don't have any issues, but when I was spot checking some old ones, I'm seeing that the Tolerant totals are not being calculated correctly.  

Production Site: http://streamwebs.org/sites/cedar-creek/macro/530/
Staging: http://streamwebs-staging.osuosl.org/sites/cedar-creek/macro/530/

The correct total should be "2".  Each area that has a count gets a value of 1, regardless of the number counted.

Here is one in staging with the same data entered and correct totals:
http://streamwebs-staging.osuosl.org/sites/test-site/macro/896/

**Comment by NathanTan** [comment]
It seems like the  non-tolerant totals should be calculated in this fashion as well, or is everything calculating correctly except for the tolerants?

**Comment by hertelc** [comment]
http://streamwebs-dev.osuosl.org:8000/sites/cedar-creek/macro/530/
The Count for "Tolerant" should be 2

Check out the paper sheet for how the numbers are calculated.
http://streamwebs.org/media/assets/Macros_OSU.pdf

**Comment by hertelc** [comment]
I am still seeing the double count on the Tolerant totals.  
http://streamwebs-dev.osuosl.org:8000/sites/wenaha-river/macro/163/

**Comment by CarvellWakeman** [comment]
When the test data script (get_all.sh) on our local dev environments runs, it ends up calling the save method for the macro model. If the model has the fix in place, this updates all old test data to be correct.

I ran the system without the model fix, and reproduced the broken data. I then wrote a migration and ran it on its own, which fixed the old data, but new data was still broken.

I applied the model fix, and that fixed new data.
